Tests/OTelTests/OTelCoreTests/Tracing/Processing/OTelSimpleSpanProcessorTests.swift

```diff
@@ -61,7 +66,13 @@ final class OTelSimpleSpanProcessorTests: XCTestCase {
         let exporter = OTelInMemorySpanExporter()
         let processor = OTelSimpleSpanProcessor(exporter: exporter)
 
-        try await processor.shutdown()
+        try await withThrowingTaskGroup { group in
+            let serviceGroup = ServiceGroup(services: [processor], logger: ._otelDebug)
+            group.addTask { try await serviceGroup.run() }
+            try await Task.sleep(for: .seconds(0.1))
```

Comment:
Can we please add a comment here explaining why we need the 0.1 seconds?

Reply:
I've been on a voyage of discovery with `ServiceGroup`. I think there's a deterministic way I can remove it.